source/guides/network/connecting-to-mongodb.rst

```diff
@@ -0,0 +1,124 @@
+.. post:: 13 Feb, 2020
+    :category: Network
+    :excerpt: 1
+    :nocomments:
+
+#########################
+How to connect to MongoDB
+#########################
+
+Learn how to access your API node MongoDB instance.
+
+**********
+Background
+**********
+
+The :doc:`REST Gateway <../../api>` offers a broad range of endpoints so that you don't have to connect to MongoDB. Still, if you are developing new plugins for Symbol, or you are analyzing extensive blockchain data, you might be looking to connect MongoDB directly.
```

Comment:
The :doc:`REST Gateway <../../api>` offers a broad range of endpoints so that you don't have to connect to MongoDB. Still, if you are developing new plugins for Symbol or analyzing extensive blockchain data, you might want to consider connecting to MongoDB directly.